bgpd/rfapi/rfapi_vty.c

```diff
@@ -3317,7 +3317,7 @@ rfapiDeleteLocalPrefixesByRFD (struct rfapi_local_reg_delete_arg *cda,
       rfapiQprefix2Rprefix (pPrefix, &rprefix);
     }
 
-  while (noop--)                /* to preserve old code structure */
+  while (loop--)                /* to preserve old code structure */
```

Comment:
Get rid of loop/noop and use a do { ....} while (0); structure?  So we know it's supposed to be run one time?